airbyte-webapp/src/views/Connector/ConnectorForm/components/Property/Control.tsx

```diff
@@ -66,6 +69,26 @@ export const Control: React.FC<ControlProps> = ({
     );
   }
 
+  if (
+    property.type === "string" &&
+    (property.format === "date-time" || property.format === "date") &&
```

Comment:
Something I noticed when testing locally was that the `End Date` field in the Facebook Marketing connector did not have a datepicker rendered, and that's because the `end_date` field doesn't have this `format` field set in the spec: https://github.com/airbytehq/airbyte/blob/5354b54c86d4ea12ce4848f5ccdabe0817f1eb79/airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json#L22

So that should be fixed (in a separate PR)

Reply:
This is not just that field. Quiet a few of connector specs are missing the `format` for date fields. We should try to have all specs double checked and fixed once this is in.